public/js/main.js

```diff
@@ -101,7 +101,7 @@ HacktoberfestChecker.prototype.makeSpinner = function() {
  */
 HacktoberfestChecker.prototype.makeError = function(error) {
     return $('<div/>').append(
-        $('<h2/>', {
+        $('<h2 class="orange"/>', {
```

Comment:
This has since been changed to be white, so can probably lose this?

Reply:
Did not mean to include this in this PR. Seems to have worked out anyway, my bad nevertheless. :)